docs/development/getting_started_locally.md

```diff
@@ -27,6 +27,10 @@ The Gardener components, however, will be run as regular processes on your machi
     ```
 
     and reload the terminal.
+- Clone the Gardener repository from GitHub.
+  ```
+  git clone https://github.com/gardener/gardener.git
```

Comment:
This prerequisite is already specified in https://github.com/gardener/gardener/blob/master/docs/development/local_setup.md#get-the-sources (along with quite other local setup related tools). IMO it does not make sense to have the same prerequisite in multiple places.

Reply:
After reviewing both documents, I have decided that while there is no need to add a duplicate prerequisite, there should be a link to the Local Setup guide. Thus there will be no duplication of prerequisites and the user will know that they need to complete the Local Setup guide before attempting to follow the Getting Started Locally tutorial.